microsoft-edge/devtools-guide-chromium/css/print-preview.md

```diff
@@ -22,23 +22,33 @@ ms.date: 05/04/2021
    limitations under the License.  -->
 # Force print preview mode
 
-The [print media query](https://developer.mozilla.org/docs/Web/CSS/Media_Queries/Using_media_queries) controls how a page looks when printed.  To force a page into print preview mode:
+The [CSS print media query](https://developer.mozilla.org/docs/Web/CSS/Media_Queries/Using_media_queries) allows you to define CSS styles that only apply when a page is printed.
 
-1. Press **Ctrl+Shift+P** (Windows, Linux) or **Command+Shift+P** (macOS) to open the **Command Menu**.
+Use the **Rendering** tool to force a webpage into print preview mode in order to debug your print styles.
 
-   > [!div class="mx-imgBorder"]
-   > ![Opening the command menu](./print-preview-images/print-preview-open-command-menu.png)
+To force a page into print preview mode:
 
-1. Type `rendering`, select **Show Rendering**, and then press **Enter**.
+1. In DevTools, open the **Command Menu**. To open the **Command Menu**, press **Ctrl+Shift+P** (Windows, Linux) or **Command+Shift+P** (macOS).
```

Comment:
@captainbrosset - the Command Menu way of doing tasks should not be the first/main way doc'd.  I plan to push a commit to doc a GUI/WIMP way as the main/first way.